go/vt/logutil/logger.go

```diff
@@ -246,6 +247,12 @@ func (ml *MemoryLogger) Clear() {
 	ml.mu.Unlock()
 }
 
+func (ml *MemoryLogger) LogEvents() []*logutilpb.Event {
```

Comment:
Nit, but IMO `GetEvents()` is a better name. If it's not safe to access Events directly, then it may also make sense to make it private (rename it to events).